src/pages/log.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBInput,
+  MDBBtn,
+  MDBIcon,
+  MDBModalFooter,
+} from 'mdbreact';
+import './log.scss';
+
+function FormPage() {
+  return (
+    <MDBContainer>
+      <MDBRow>
+        <MDBCol md="5" className="mt-5">
+          <MDBCard className="mt-5 " className="log">
```

Comment:
If you want to have two class names on the same Tag do this
```suggestion
          <MDBCard className="mt-5 log">
```